app/services/github_repository_client.rb

```diff
@@ -0,0 +1,15 @@
+class GithubRepositoryClient
+  URL = 'https://api.github.com/repos/rootstrap'.freeze
+
+  def initialize(project_name)
+    @project_name = project_name
+  end
+
+  def get_content_from_file(file_name)
+    connection = Faraday.new(url: "#{URL}/#{@project_name}/contents/#{file_name}") do |conn|
+      conn.basic_auth(ENV['GITHUB_ADMIN_USER'], ENV['GITHUB_ADMIN_TOKEN'])
+    end
+    response = connection.get { |req| req.headers['Accept'] = 'application/vnd.github.v3.raw' }
+    response.success? ? response.body : ''
+  end
```

Comment:
Shouldn't this be a private method?

Reply:
if this method becomes private we lose the entry point of the class, we don't have a `call` method because the methods of this class should be the different endpoints of the github repos API